package.json

```diff
@@ -21,7 +21,7 @@
     "serverless",
     "ssr"
   ],
-  "version": "0.14.1",
```

Comment:
You shouldn't change this or the Changelog manually, it will be done by automation during release.